.github/workflows/ci.yml

```diff
@@ -12,6 +12,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        build_flags:
+          - ""
+          - " -- --enable-bundled-libffi"
```

Comment:
Could you add this information to `name`?
We can't distinct this information from the current job list...

* macos-10.15 debug: https://github.com/ruby/fiddle/pull/94/checks?check_run_id=3701967650
* macos-10.15 debug: https://github.com/ruby/fiddle/pull/94/checks?check_run_id=3701968329